### PR TITLE
Current and Future high level features list

### PR DESCRIPTION
**Pull request by gunamata**
This document lists Rancher Desktop high level features and their current implementation status.

### CODE REVIEW

**Comment by agracey** [comment]
It might be better to put these all in a table with a column for each OS/arch target combo (windows x86, mac x86, mac M1, linux, etc...)

**Comment by mook-as** [comment]
Should this live here or in the [docs repo](https://github.com/rancher-sandbox/docs.rancherdesktop.io)?

File: docs/development/features.md [hunk 1]
```diff
@@ -0,0 +1,151 @@
...
+
+### IDE compatibility
+
+:heavy_check_mark: VS Code extension (Remote Containers)
+
+:sun_with_face: Visual Studio IDE (Needs Validation)
+
+:sun_with_face: Eclipse (Needs Validation)
+
+### Other Rancher projects integration
```
Comment (ericpromislow) [inline docs/development/features.md:133]: I'd suggest: `Integration with Other Rancher Projects`

**Review by ericpromislow — CHANGES_REQUESTED** [review]
Mostly a bunch of stylistic quibbles to increase consistency

File: docs/development/features.md [hunk 2]
```diff
@@ -0,0 +1,151 @@
...
+This document lists the high level Rancher Desktop features and their current status.
+
+| Symbol | Description |
+| ------------- | ---------------- |
+| :heavy_check_mark: | released |
+| :calendar: | targeted for the [next] or the [later] milestone release |
+| :sun_with_face:| not planned yet, but considering for a future release |
+
+Note:
+- Items under [next] milestone are targeted for the upcoming monthly release, which usually happens on the 4th wednesday of the month.
```
[the remark this answers is not in the document]
Comment (gunamata) [reply docs/development/features.md:12]: Updated per your suggestion.

**Comment by gunamata** [comment]
Updated based on grammar, language suggestions by @ericpromislow .

File: docs/development/features.md [hunk 3]
```diff
@@ -0,0 +1,151 @@
+# Rancher Desktop Features
+
+This document lists the high level Rancher Desktop features and their current status.
```
Comment (ericpromislow) [inline docs/development/features.md:3]: `high-level`

**Review by ericpromislow — CHANGES_REQUESTED** [review]
Sorry, I missed that other one. Everything else looks good.

File: docs/development/features.md (hunk 3 above)
Comment (gunamata) [reply docs/development/features.md]: @ericpromislow , I changed to `high-level` per your suggestion.

File: docs/development/features.md [hunk 4]
```diff
@@ -0,0 +1,151 @@
...
+
+:heavy_check_mark: Mac (Intel)
+
+:heavy_check_mark: Mac M1 (apple silicon)
+
+:heavy_check_mark: Linux
+
+:sun_with_face: Linux arm64
+
+:sun_with_face: Windows on arm
```
Comment (ericpromislow) [inline docs/development/features.md:31]: Windows on AArch64 (or `AArch32` if you actually mean the 32-bit variant).

File: docs/development/features.md [hunk 5]
```diff
@@ -0,0 +1,151 @@
...
+
+:heavy_check_mark: Win 10/11
+
+:heavy_check_mark: Mac (Intel)
+
+:heavy_check_mark: Mac M1 (apple silicon)
+
+:heavy_check_mark: Linux
+
+:sun_with_face: Linux arm64
```
Comment (ericpromislow) [inline docs/development/features.md:29]: Linux AArch64

**Review by ericpromislow — CHANGES_REQUESTED** [review]
Still noticing things: the 'w' in Wednesday should be capitalized.

According to https://documentation.suse.com/style/current/single-html/docu_styleguide/#sec-terminology `arm64` should be called `AArch64`.
The doc doesn't mention `arm`, but I would use `AArch32` until I hear from them otherwise.

**Review by ericpromislow — APPROVED** [review]
Looks good

**Pull request merged**